ansible/local_testing/README.md

```diff
@@ -55,7 +55,6 @@ vms:
 - lovelace: 192.168.56.3
 - neumann: 192.168.56.4
 - richie: 192.168.56.5
-- turing: 192.168.56.6
```

Comment:
This is probably a good time to update the whole group here to reflect the new removed hosts.